### ISSUE TITLE
layout reset

### ISSUE THREAD

**Issue by shaiksaberali** [spyder-ide/spyder#6403]
## Description

**What steps will reproduce the problem?**

1. 
2. 
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "C:\Users\user\Anaconda3\lib\site-packages\spyder\app\mainwindow.py", line 1812, in reset_window_layout
    self.setup_layout(default=True)
  File "C:\Users\user\Anaconda3\lib\site-packages\spyder\app\mainwindow.py", line 1455, in setup_layout
    self.save_current_window_settings(prefix, section)
  File "C:\Users\user\Anaconda3\lib\site-packages\spyder\app\mainwindow.py", line 1415, in save_current_window_settings
    self.maximize_dockwidget(restore=True)# Restore non-maximized layout
  File "C:\Users\user\Anaconda3\lib\site-packages\spyder\app\mainwindow.py", line 2268, in maximize_dockwidget
    self.last_plugin.dockwidget.setWidget(self.last_plugin)
AttributeError: 'NoneType' object has no attribute 'dockwidget'

## Version and main components

* Spyder Version: 3.2.4 
* Python Version: 3.6.3
* Qt Versions: 5.6.2, PyQt5 5.6 on Windows

## Dependencies
```
pyflakes >=0.6.0 :  1.6.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.20.3 (OK)
numpy >=1.7      :  1.13.3 (OK)
sphinx >=0.6.6   :  1.6.3 (OK)
rope >=0.9.4     :  0.10.5 (OK)
jedi >=0.9.0     :  0.10.2 (OK)
nbconvert >=4.0

**Comment by ccordoba12** [comment]
This was fixed in our 3.2.6 version. Please update.

**Comment by CAM-Gerlach** [comment]
For the record, duplicate of #5477 . You can update with ``conda update spyder`` from the Anaconda prompt.
